apps/condo/domains/user/schema/User.js

```diff
@@ -180,6 +180,13 @@ const User = new GQLListSchema('User', {
             graphQLReturnType: 'CustomAccess',
             graphQLInputType: 'CustomAccessInput',
         },
+
+        showGlobalHints: {
+            schemaDoc: 'Show global hints in CRM or not',
```

Comment:
Will there be hints other than CRM (mb in mobile app)?

Reply:
Currently only in CRM

Reply:
We've got a `features` field in Organization model, which is integrated with growthbook. Can it be used here?

Reply:
This is a for every user setting, not for organizations